### PR TITLE
Configurable: remove unused double foreach loop

### PR DESCRIPTION
**Pull request by Hanmac**
### Description (*)

The double `foreach` does nothing of value inside the Block:
- it just calls `$this->_preparePrice` which just does format the price value.
- but then it does nothing with it. Even where it supposed to go inside `$_prices` variable hasn't been working for 16 years since Magento 1.1 when this file was created.
- In theory, the variables aren't even used and could be removed too. But that might be more of a BC break.

### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#2246

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

File: app/code/core/Mage/Catalog/Block/Product/View/Type/Configurable.php [hunk 1]
```diff
@@ -25,14 +25,14 @@ class Mage_Catalog_Block_Product_View_Type_Configurable extends Mage_Catalog_Blo
...
-     *
+     * @deprecated
      * @var array
      */
     protected $_prices      = [];
 
     /**
      * Prepared prices
-     *
+     * @deprecated
```
[the remark this answers is not in the document]
Comment (Hanmac) [reply app/code/core/Mage/Catalog/Block/Product/View/Type/Configurable.php:35]: but only is set, it is never used (that's why i didn't removed the variables yet)

[the remark this answers is not in the document]
Comment (fballiano) [reply app/code/core/Mage/Catalog/Block/Product/View/Type/Configurable.php:35]: @Hanmac true, should the _preparePrice call be removed too?

[the remark this answers is not in the document]
Comment (Hanmac) [reply app/code/core/Mage/Catalog/Block/Product/View/Type/Configurable.php:35]: I think yeah, but that might be for later MR

**Review by sreichel — APPROVED** [review]
There is no variable assigned. Parameters arent used as reference.

Dont know to tes, but it looks useless.

**Review by kiatng — APPROVED** [review]

**Pull request merged**